plugins/search/providers/elastic_search/lib/kElasticSearchManager.php

```diff
@@ -13,6 +13,7 @@ class kElasticSearchManager implements kObjectReadyForIndexEventConsumer, kObjec
     const CACHE_PREFIX_STICKY_SESSIONS = 'elastic_large_sql_lock_';
     const REPETITIVE_UPDATES_CONFIG_KEY = 'skip_elastic_repetitive_updates';
     const METADATA_MAX_LENGTH = 256;
+	const SQL_MAX_ALLOWED_PACKET = 1048576; // 1MB (1048576 / 1024 / 1024)
```

Comment:
indent

Reply:
Good point - noted.
For now we're not handling it since it's fairly rare and only impact the specific object
Also, per Yossi comment above, when we will proceed with this fix I'll make it after we compress the data